subxt/src/config/refine_params.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! Refining params with values fetched from the chain
+
+use crate::Config;
+
+/// Types implementing [`RefineParams`] can be modified to reflect live information from the chain.
+pub trait RefineParams<T: Config> {
```

Comment:
Nit: I'd call this `RefinableParams` (because then you read `Params: RefinableParams<T>` which says.. these params are refinable!) and then call method `refine_params` I reckon so it's more obvious it relates to the trait

Reply:
Hmm, I also like the name with the _able_, but I thought in rust it is generally a convention to about that, e.g. `Clone` instead of `Clonable`, `Serialize` instead of `Serializable`,[ etc.](https://github.com/rust-lang/api-guidelines/discussions/28)

Reply:
Hmm true! Maybe it's better to leave it as is then, just for consistency :)